### PR TITLE
#164046202 Setup Redux

### PR DESCRIPTION
**Pull request by ElMonstro**
### What does this PR do?
It sets up redux
### Description
It adds:
 - store
 - root reducer
 - redux-thunk middleware

### PT
[#164046202](https://www.pivotaltracker.com/story/show/164046202)

### CODE REVIEW

**Review by BrianSerem — APPROVED** [review]

**Review by MandelaK — APPROVED** [review]

**Review by MandelaK — COMMENTED** [review]
Could you please contain directories for `components` and `containers`, `reducers` and `actions` in this PR?

**Review by RachelleMaina — APPROVED** [review]
Looks good.

**Comment by ElMonstro** [comment]
@MandelaK I cannot add that because git does not track directories. Team members will create those directories when needed.

**Comment by MandelaK** [comment]
> @MandelaK I cannot add that because git does not track directories. Team members will create those directories when needed.

Ahahaha, yea ... my bad ... Looks good to me

File: src/store/store.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+const storeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+
+const store = createStore(
+    rootReducer, 
+    storeEnhancers(applyMiddleware(thunk)),
+    );
+
+
+export default store
```
Comment (abulojoshua1) [inline src/store/store.js]: @ElMonstro, kindly add an EOF here.

**Review by abulojoshua1 — CHANGES_REQUESTED** [review]

**Review by Etomovich — APPROVED** [review]

File: src/index.js [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
+    <Provider store={ store }>
+        <App />
+    </Provider>,
+        document.getElementById('root')
+    );
+    
+    // If you want your app to work offline and load faster, you can change
+    // unregister() to register() below. Note this comes with some pitfalls.
+    // Learn more about service workers: https://bit.ly/CRA-PWA
+    serviceWorker.unregister();
```
Comment (cob04) [inline src/index.js]: I don't know whether using `service workers` is really necessary here. If after relooking you decide it is not, I suggest you remove it.

**Review by cob04 — APPROVED** [review]
looks ok

File: src/store/store.js (hunk 1 above)
Comment (ElMonstro) [reply src/store/store.js]: @abulojoshua1  I have done the requested change

**Review by abulojoshua1 — APPROVED** [review]
LGTM

**Pull request merged**